src/stories/components/custom-table/row-item/item-core-unit.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import { columns, CoreUnit } from '@ses/core/utils/test.utils';
+import ItemCoreUnit from './item-core-unit';
+import type { ComponentMeta } from '@storybook/react';
+
+export default {
+  title: 'Components/CUTable/ItemCoreUnit',
+  component: ItemCoreUnit,
+  parameters: {
+    fullscreen: 'centered',
+    chromatic: {
+      viewports: [1194, 1280],
+      pauseAnimationAtEnd: true,
+    },
+  },
+  decorators: [
+    (Story) => (
+      <div
+        style={{
+          margin: '3em',
+          width: 'fit-content',
+        }}
+      >
+        <Story />
+      </div>
+    ),
+  ],
+} as ComponentMeta<typeof ItemCoreUnit>;
+
+const variantsArgs = [
+  {
+    queryStrings: '',
+    key: '2',
```

Comment:
the key should be job of the component, specifically inside the loop based on the CU data

Reply:
you are right